modules/react/button/lib/parts/ButtonContainer.tsx

```diff
@@ -96,7 +96,7 @@ export const ButtonContainer = styled('button', {
     // Makes the "down" state of the button happens faster than the hover state, so it animates in correctly.
     '&:hover:active': {transitionDuration: '40ms'},
 
-    '&:disabled, &:disabled:active': {cursor: 'default', boxShadow: 'none'},
+    '&:disabled, &:disabled:active': {cursor: 'default', boxShadow: 'none', opacity: 0.4},
```

Comment:
thinking out loud, if we define this opacity and it controls the opacity of all of our buttons, should we define this at a higher level and name is some variable like `buttonOpacity`, maybe not worth the effort...

Reply:
I think it's a fair call, if for no other reason to put a comment with some context around it. 👍🏻